docs/_build/html/_sources/methods.rst.txt

```diff
@@ -0,0 +1,444 @@
+Index in `xolotl` tree  ``tree_idx``
+Index of neuron         ``neuron_idx``
+Length of compartment   ``len``
+Ca++ buff. time const.  ``tau_Ca``
+Membrane capacitance    ``Cm``
+Intra. calcium conc.    ``Ca``
+Membrane potential      ``V``
+======================= ================
+
+To add a compartment named ``HH`` with a membrane capacitance ``Cm = 10`` ::
+
+  x.add('HH', 'compartment', 'Cm', 10);
+
+To access properties of that compartment, use the syntax ``x.HH``. To add a conductance to that compartment ::
+
+  x.HH.add('ConductancePath', 'PropertyName', 'PropertyValue', ...);
+
+``ConductancePath`` is the path to the ``.hpp`` file inside ``xolotl/c++/conductances/``. For example, the conductance path to add the slow calcium conductance from Prinz *et al.* 2003 is ``prinz/CaS`` since the conductance is specified in ``xolotl/c++/conductances/prinz/CaS.hpp``.
+
+The following properties can be specified
+
+======================= ================
+Name                    ``PropertyName``
+Maximal conductance     ``gbar``
+Reversal potential      ``E``
+Inactivation variable   ``h``
+Activation variable     ``m``
+======================= ================
+
+For example ::
+
+  x.HH.add('prinz/CaS', 'gbar', 500, 'E', -80);
+
+The maximal conductance defaults to 0, the reversal potential defaults based on
+the conductance. Activation variables (if any) default to zero. Inactivation variables
+default to one. Synapses can be added this way, but it is better to use the ``connect`` method.
+
+.. _cleanup:
+
+cleanup
+^^^^^^^
+
+When the ``C++`` code compiles, it produces a binary ``.mexa64`` file and a ``.cpp`` file in your ``xolotl`` directory. These files are hashed so that repeated simulation does not require recompilation. If these files are too numerous or broken, you can erase them all by calling either ::
+
+  xolotl.cleanup
+  x.cleanup
+
+
+.. _compile:
+
+compile
+^^^^^^^
+
+Hashes and compiles the files needed to run a simulation. These are stored in
+your ``xolotl`` directory. ``xolotl`` automatically compiles when it needs to.
+You can turn this functionally off by setting ::
+
+  x.skip_hash = true;
+
+In addition, creating a ``xolotl`` object through a function call does not
+automatically hash and compile. In this case, you should use ``x.md5hash``.
+=======
+In addition, creating a ``xolotl`` object through a function call does not automatically hash and compile. In this case, you should use ``x.md5hash``.
+>>>>>>> 2c44bd81aed5d11aac7fb1677a4b462d47822744
+
+.. hint::
+
+  Always transpile before you compile! ::
+
+    x.transpile;
+    x.compile;
+
+.. _connect:
+
+connect
+^^^^^^^
+
+Connects two compartments with a synapse. The basic syntax is ::
+
+  x.connect('PreSynaptic', 'PostSynaptic', 'Type', PropertyName', PropertyValue, ...)
+
+The first two arguments are the presynaptic and postsynaptic compartment names.
+
+In the case of two arguments, or three arguments where the third is a ``double``,
```

Comment:
a better way of saying this is : connect defaults to using an Axial synapse if...